crates/rolldown/src/module_loader/runtime_ecma_module_task.rs

```diff
@@ -114,21 +120,21 @@ impl RuntimeEcmaModuleTask {
       std::mem::take(&mut symbol_table.references),
       std::mem::take(&mut symbol_table.resolved_references),
     );
-    let mut symbol_for_module = AstSymbols::from_symbol_table(symbol_table);
+    let mut ast_symbols = AstSymbols::from_symbol_table(symbol_table);
     let facade_path = ResourceId::new("runtime");
     let scanner = AstScanner::new(
       self.module_id,
       &ast_scope,
-      &mut symbol_for_module,
+      &mut ast_symbols,
       "runtime".to_string(),
       ModuleDefFormat::EsmMjs,
       source,
       &facade_path,
       &ast.trivias,
     );
-    let namespace_symbol = scanner.namespace_object_ref;
+    let namespace_object_ref = scanner.namespace_object_ref;
     let scan_result = scanner.scan(ast.program());
 
-    Ok((ast, ast_scope, scan_result, symbol_for_module, namespace_symbol))
```

Comment:
Using a tuple with the length of 5 is a bad idea, Adding a temp struct to store the result.